skipad.js

```diff
@@ -8,21 +8,25 @@
    * The interval period of 2 seconds is arbitary, but I guess 2sec
    * is a good choice.
    */
-  var timeout = setInterval(function() {
+  var timeout = setInterval(function () {
     if (window.location.pathname !== '/watch') {
       return;
     }
-    var skipButton = document.getElementsByClassName('videoAdUiSkipButton')[0];
+    var skipButton = document.getElementsByClassName('ytp-ad-skip-button ytp-button')[0];
```

Comment:
I believe YouTube is rolling out these changes gradually since the class names have not changed for me so far. So changing the class name like this will break the extension for a lot of the users.

We should probably have a list of possible class/ids that should be checked.